test/onnx/test_pytorch_common.py

```diff
@@ -76,5 +76,17 @@ def wrapper(self):
         return wrapper
     return skip_dec
 
+# skips tests that do not follow ONNX IR v3 (old-style) export.
+# Only tests that follow ONNX IR v3 export, i.e. where 
+# keep_initializers_as_inputs is set to True in torch.onnx.export API call.
+def skipIfNotOnnxIRv3():
```

Comment:
This particular test is skipped for ONNX IRv4 flavors, because ORT optimizer has a bug in loading this model for IRv4 format (not in IRv3 format). As soon as that issue is resolved, I will enable this test.